package.json

```diff
@@ -28,40 +46,22 @@
     "uuid": "^9.0.0",
     "v-tooltip": "2.1.3",
     "vue-click-outside": "^1.1.0",
+    "vue-loader": "^15",
     "vue-multiselect": "^2.1.6",
     "vue-omnibox": "^0.3.7",
     "vue-sliding-pagination": "^1.3.2",
+    "vue-template-compiler": "^2.7.14",
     "vuedraggable": "^2.24.3",
     "vuex": "^3.6.2",
-    "@babel/core": "^7.20.2",
-    "@babel/preset-env": "^7.20.2",
-    "@storybook/addon-actions": "6.5.13",
-    "@storybook/addon-docs": "6.5.13",
-    "@storybook/addon-essentials": "6.5.12",
-    "@storybook/addon-links": "6.5.12",
-    "@storybook/vue": "6.5.12",
-    "@webpack-cli/generators": "^3.0.0",
-    "autoprefixer": "^10.4.13",
-    "babel-loader": "^9.1.0",
-    "babel-plugin-syntax-jsx": "^6.18.0",
-    "babel-plugin-transform-vue-jsx": "^3.7.0",
-    "css-loader": "^6.7.2",
-    "js-beautify": "^1.14.7",
-    "mini-css-extract-plugin": "^2.7.0",
-    "postcss": "^8.4.19",
-    "postcss-loader": "^7.0.1",
-    "style-loader": "^3.3.1",
-    "vue-loader": "^15",
-    "vue-template-compiler": "^2.7.14",
     "webpack": "^5.75.0",
     "webpack-cli": "^5.0.0"
   },
   "peerDependencies": {
+    "react": "16.14.0",
```

Comment:
Don't have to be done right now I You are uncertain.
may react be a devDependency for Storybook. Or may we drop it? I know. I added it here....